templates/about.html

```diff
@@ -23,10 +23,8 @@ <h2>About Monadical</h2>
                 Read more about our <a href="https://docs.monadical.com/s/principles-handbook">company principles</a> or <a href="https://docs.monadical.com/s/monadical-study-guide">onboarding process</a>.
                 <br/><br/><br/>
                 <center>
-                    <img src="static/team.jpg" style="width: 50%; border-radius: 14px"/>
+                    <img src="static/teamretreat.jpg" style="width: 100%; border-radius: 14px"/>
```

Comment:
I suggest to keep the name and replace the file to avoid archiving files in the repo